src/main/java/com/strandls/authentication_utility/util/AuthUtil.java

```diff
@@ -122,6 +125,41 @@ public static void updateUserProfile(CommonProfile profile, Long userId, String
 		return response;
 	}
 
+	public static Map<String, Object> generateAccessTokenFromEmail(String email) {
+		Map<String, Object> response = new HashMap<>();
+		if (!email.isEmpty() && email != null) {
+
+			JwtGenerator<CommonProfile> generator = new JwtGenerator<>(
+					new SecretSignatureConfiguration(PropertyFileUtil.fetchProperty(CONFIGURATION, JWT_SALT)));
+
+			Map<String, Object> jwtClaims = new HashMap<>();
```

Comment:
Would it be possible to have jwtClaims as a POJO instead of a hashmap?

Reply:
the jwtClaims being dynamic in nature (varies accord. to the use case), i think it would make more sense if we keep it as a object

Reply:
Ok